### PR TITLE
New module for a Secure Delete

### PR DESCRIPTION
**Pull request by jbenden**
New formula for a secure delete by THC.org.

### CODE REVIEW

**Comment by tdsmith** [comment]
Mike said earlier:

> Yeh, this [patch] will need to be submitted upstream before we can merge.

That's still the case. Once that's done, please add a Ruby comment explaining that it's been submitted, ideally with a URL to a mailing list post or an issue ticket or something similar. Thanks!

**Comment by jacknagel** [comment]
ping

**Comment by jbenden** [comment]
Is there something that I need to do to get this patch submitted? Thanks!

File: Library/Formula/srm.rb [hunk 1]
```diff
@@ -0,0 +1,76 @@
+require "formula"
+
+class Srm < Formula
+  homepage "http://www.thc.org/"
+  url "https://www.thc.org/releases/secure_delete-3.1.tar.gz"
+  sha1 "ce6391003587b37bd3713a967de04de1f0759107"
+
+  keg_only :provided_by_osx if MacOS.version >= :mavericks
+
+  patch :DATA
```
Comment (mikemcquaid) [inline Library/Formula/srm.rb]: Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!

**Comment by mikemcquaid** [comment]
Please fix the errors reported by `brew audit --strict srm`, thanks!

**Comment by jbenden** [comment]
I updated to the latest homebrew, but I'm not getting any errors by running the audit command. Can someone paste in the output of running that command so I may fix those errors. I would really like to square away this patch - it's actually useful. Thanks!

**Comment by DomT4** [comment]
Are you running the audit command with ` --strict ` invoked?

**Comment by mikemcquaid** [comment]
@BrewTestBot test this please

**Comment by jbenden** [comment]
Yes, I am running it with --strict. No output is seen.

**Comment by DomT4** [comment]
```
Error: 2 problems in 1 formulae

==> audit problems
srm:
 * Stable: SHA1 checksums are deprecated, please use SHA256
 * `require "formula"` is now unnecessary
```

**Comment by jbenden** [comment]
I cleaned it up. Please review. Thanks!

File: Library/Formula/srm.rb [hunk 2]
```diff
@@ -0,0 +1,74 @@
+class Srm < Formula
+  homepage "http://www.thc.org/"
+  url "https://www.thc.org/releases/secure_delete-3.1.tar.gz"
+  sha256 "a9d846d1dce3f1bdf13bbb306e8596bc1f263198a086f6beecd90ccf7bddf8d5"
+
+  keg_only :provided_by_osx if MacOS.version >= :mavericks
+
+  patch :DATA
```
Comment (mikemcquaid) [inline Library/Formula/srm.rb]: Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!

**Comment by jbenden** [comment]
I cannot agree to submit the patch upstream as the changes are specific to homebrew only. It is my opinion that they remain in homebrew as a patch.

**Comment by DomT4** [comment]
What part of the patch have you applied here that wouldn't be applied for a normal OS X user wishing to run this software, out of curiosity?

**Comment by mikemcquaid** [comment]
@jbenden We don't accept software that requires unmergeable patches to work on OS X.

**Comment by mikemcquaid** [comment]
And generally we don't accept new software that requires patched at al.

**Comment by jacknagel** [comment]
Passing on this unless the above is addressed.

**Pull request closed without merging**